README.md

```diff
@@ -1 +1,129 @@
-# OOP-school-library
\ No newline at end of file
+## 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [School Library App] <a name="about-project"></a>
+
+**[School Library App]**  As a librarian at OOP University, I was tasked with developing a tool to manage the library's book inventory and keep track of borrowers. The project involves designing an application that will enable uses to:
+- Register new students and teachers into the system.
+- Catalog new books as they are added to the library.
+- Maintain a comprehensive record of individuals who have borrowed specific books and the corresponding borrowing dates.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Create student]**
+- **[Create Teacher]**
+- **[Create books]**
+- **[Keep records of borrowed items]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+-postgres installed
+-Laptop
+-Internet
+-Github
+-Command Line
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ctokoli/OOP-school-library.git
+```
+
+### Install
+
+Install this project with:
+
+Install Ruby [https://www.ruby-lang.org/en/documentation/installation/]
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Cephas, Tokoli**
+
+- GitHub: [@Ctokoli](https://github.com/ctokoli/)
+- Twitter: [@Ctokoli](https://twitter.com/ctokoli)
+- LinkedIn: [Cephas Tokoli](https://www.linkedin.com/in/cephas-tokoli-a25204123/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
```

Comment:
- [OPTIONAL] It would be nice to remove this placeholder text from this section of the README as you did in other sections of the README. The placeholder texts just describe what should be done in each section of the README. 👍